### DISCUSSION TITLE
List of ideas

### DISCUSSION THREAD

**Discussion by SiIentNinja** [ScratchAddons/ScratchAddons#6805] [Ideas] [answered]
HI!
I've been collecting ideas over the past year, adding them to the list whenever I come up with one, or feel an inconveniency with Scratch.
Disclaimer: Because this list was made over a long time, some of these features might have been implemented by the time this is posted.
These are just suggestions, feel free to change them however you want, and I won't be surprised If you don't add any:)

More fonts in art editor, because the normal ones aren't good enough for most situations
Opacity slider in art editor (Like turbowarp)
show "if touching [own sprite name]" option (clone detection). Can already do this by dragging sprite detection from a different sprite
focus mode- hide trending, only see own projects, etc. (to help focus on making own projects), I really need this lol
see how long spent coding a project- great for time management, and interesting to see
see amount of blocks used in a project- the "amount of scripts used" doesn't always demonstrate the actual size
pin projects to top of my stuff- Useful if working on multiple projects, or want to find something after a while without needing to scroll down a lot
fps counter and project timer at top (like the clone counter)- useful for debugging
make custom blocks "for all sprites" option
rearrange scratch projects in profile, allows for priority of better projects (without needing to reshare in quality order)
Search projects in another users' list of projects- useful for searching for projects that will not be found easily with normal search
show working hacked blocks (eg while, for and when touching hat block)- more convenient than backpack
"find" feature can search every type of block. It's annoying that it can only search some, I often find myself looking for a block that isn't in the search.
And lastly-
turn move_steps into move_step if it is 1 (same with other movement blocks like turn_degrees), cause it hurts my brain

Just some suggestions, but I hope these will inspire you to add something like one of my ideas to Scratch addons :)

**Comment by WorldLanguages** [comment, marked as answer]
> fps counter and project timer at top (like the clone counter)- useful for debugging

Looks like #6807 has been recently opened.

> rearrange scratch projects in profile, allows for priority of better projects (without needing to reshare in quality order)

Impossible, but I always wanted this

> Search projects in another users' list of projects- useful for searching for projects that will not be found easily with normal search

Could be useful!
